test/powershell/Modules/Microsoft.PowerShell.Utility/Out-String.Tests.ps1

```diff
@@ -45,4 +45,14 @@ Describe "Out-String" -Tags "CI" {
 
 	$streamoutputlength | Should BeLessThan $nonstreamoutputlength
     }
+
+    It "Should not print a newline when the nonewline switch is used" {
+        $testArray = "a", " b"
```

Comment:
Why we use space? I'd prefer to remove.

Reply:
I was not sure what example to give and hence followed the one in 
It "Should accumulate the strings and returns them as a single string"
Will remove the space as suggested

Reply:
removed the space